doc/cygnus-common/installation_and_administration_guide/management_interface_v1.md

```diff
@@ -195,6 +208,282 @@ Response:
+                            "originalEntityType":"entityType1",
+                            "newEntityId":"newEntityId1",
+                            "newEntityType":"newEntityType1",
+                            "attributeMappings":[
+                                {
+                                    "originalAttributeName":"attrName1",
+                                    "originalAttributeType":"attrType1",
+                                    "newAttributeName":"newAttrName1",
+                                    "newAttributeType":"newAttrType1"
+                                },{
+                                   ...
+                                }
+                            ]
+                        },{
+                           ...
+                        }
+                    ]
+                },{
+                   ...
+                }
+            ]
+        },{
+           ...
+        }
+    ]
+}
+```
+
+Response:
+
+```
+{
+    "success":"true",
+    "serviceMapping": [
+        {
+            "originalService":"service1",
+            "newService":"newService1",
+            "servicePathMappings": [
+                {
+                    "originalServicePath":"/servicePath1",
+                    "newServicePath":"/newServicePath1",
+                    "entityMappings": [
+                        {
+                            "originalEntityId":"entityId1",
+                            "originalEntityType":"entityType1",
+                            "newEntityId":"newEntityId1",
+                            "newEntityType":"newEntityType1",
+                            "attributeMappings":[
+                                {
+                                    "originalAttributeName":"attrName1",
+                                    "originalAttributeType":"attrType1",
+                                    "newAttributeName":"newAttrName1",
+                                    "newAttributeType":"newAttrType1"
+                                },{
+                                   ...
+                                }
+                            ]
+                        },{
+                           ...
+                        }
+                    ]
+                },{
+                   ...
+                }
+            ]
+        },{
+           ...
+        }
+    ]
+}
+```
+
+[Top](#top)
+
+###<a name="section5.3"></a>`PUT /v1/namemappings`
+Updates already existent [name mappings](../../cygnus-ngsi/flume_extensions_catalogue/ngsi_name_mappings_interceptor.md). The updated mappings are given within the request payload, following the Name Mappings Json format (for instance, if an already existent entity mapping has to be updated, the service and the service path it belongs to must be present in the Json as well). All the mappings within the payload that are present in the configuration are updated.
+
+```
+PUT http://<cygnus_host>:<management_port>/v1/namemappings
```

Comment:
In REST the typical pattern is to include the identification of the resource to modify (resource = name mapping in this case) in the URL. However, that is not done in this case (PUT).

I have check that this also happens with the DELETE method. This is even more striking, as DELETE operations in REST doesn't use to need payload...

Maybe I'm not uderstanding the nature of the name mapping API. To be discussed once you get back after holiday.

Reply:
It will be noted in issue #1227, pending on a new PR